src/components/Nav.tsx

```diff
@@ -34,45 +61,55 @@ export default function Nav() {
         marginRight: {sm: "1rem", md: "2rem", lg: "3.5rem"},
       }}
     >
-      <Button
-        variant="nav"
-        component={NavLink}
+      <NavButton
         to="/transactions"
-        title="View all Transactions"
-        sx={{
-          color: "inherit",
-          fontSize: {sm: ".875rem", md: "1rem"},
-        }}
-      >
-        Transactions
-      </Button>
-      <Button
-        variant="nav"
-        onClick={handleGovernanceClick}
-        title="Aptos Governance"
-        sx={{
-          color: "inherit",
-          fontSize: {sm: ".875rem", md: "1rem"},
-        }}
-      >
-        Governance
-      </Button>
-      <Menu
-        open={open}
-        onClose={handleClose}
-        anchorEl={governanceMenuEl}
-        TransitionComponent={Fade}
-        aria-controls={open ? "fade-menu" : undefined}
-        aria-haspopup="true"
-        aria-expanded={open ? "true" : undefined}
-      >
-        <MenuItem onClick={() => handleCloseAndNavigate("/proposals")}>
-          Proposals
-        </MenuItem>
-        <MenuItem onClick={() => handleCloseAndNavigate("/proposals/staking")}>
-          Staking
-        </MenuItem>
-      </Menu>
+        title="View All Transactions"
+        label="Transactions"
+      />
+      {inDev && (
+        <>
+          <NavButton to="/blocks" title="View All Blocks" label="Blocks" />
+          <NavButton
+            to="/Validators"
+            title="View All Validators"
+            label="Validators"
+          />
+          <NavButton to="/tokens" title="View Top Tokens" label="Tokens" />
```

Comment:
do we want to use the `MenuItem` component as we do with all the other items?
https://github.com/aptos-labs/explorer/pull/179/files#diff-7f2ef60bbec771e3e2a3e1e7162d91e481d7724cd76ea86a82a0b4ad45c3baefR102

Reply:
We want them to be buttons in the header. `MenuItem` will be used later when we have sub menu for the buttons. For example: 
+ Tokens
    + Top Tokens
    + Top Collections